newswires/client/src/sharedTypes.ts

```diff
@@ -24,3 +24,5 @@ export const WireDataSchema = z.object({
 });
 
 export type WireData = z.infer<typeof WireDataSchema>;
+
+export const WireDataArraySchema = z.array(WireDataSchema);
```

Comment:
in a future PR I plan to replace array here with object containing a results key with the array as a value, in anticipation of adding other fields to the search response for additional metadata (notably keywords count)